app/src/features/standards/view/SpeciesStandardsResults.tsx

```diff
@@ -0,0 +1,101 @@
+import { mdiRuler, mdiTag } from '@mdi/js';
+import { Box, CircularProgress, Divider, Stack, Typography } from '@mui/material';
+import { useBiohubApi } from 'hooks/useBioHubApi';
+import useDataLoader from 'hooks/useDataLoader';
+import { ITaxonomy } from 'interfaces/useTaxonomyApi.interface';
+import { useEffect, useState } from 'react';
+import MarkingBodyLocationStandardCard from './components/MarkingBodyLocationStandardCard';
+import MeasurementStandardCard from './components/MeasurementStandardCard';
+import SpeciesStandardsToolbar, { SpeciesStandardsViewEnum } from './components/SpeciesStandardsToolbar';
+
+interface ISpeciesStandardsResultsProps {
+  selectedSpecies?: ITaxonomy;
+}
+
+const SpeciesStandardsResults = (props: ISpeciesStandardsResultsProps) => {
+  const biohubApi = useBiohubApi();
+  const [activeView, setActiveView] = useState<SpeciesStandardsViewEnum>(SpeciesStandardsViewEnum.MEASUREMENTS);
+
+  const resourcesDataLoader = useDataLoader((tsn: number) => biohubApi.standards.getSpeciesStandards(tsn));
+
+  useEffect(() => {
+    if (props.selectedSpecies) {
+      resourcesDataLoader.refresh(props.selectedSpecies.tsn);
+    }
+  }, [props.selectedSpecies?.tsn]);
+
+  if (!resourcesDataLoader.data) {
+    return <></>;
```

Comment:
Empty state here would probably be nice

Reply:
What do you mean?

Reply:
In other components we have "empty state" which renders when no selection has occurred yet.
Like "No animal selected" or "No rows to show", if only we think that would be nice for this page.